sbi/neural_nets/density_estimators/base.py

```diff
@@ -36,8 +60,8 @@ def loss(self, input: Tensor, condition: Tensor, **kwargs) -> Tensor:
         r"""Return the loss for training the density estimator.
 
         Args:
-            input: Inputs to evaluate the loss on.
-            condition: Conditions.
+            input: Inputs to evaluate the loss on of shape (batch_size, d).
+            condition: Conditions of shape (batch_size, *condition_shape).
```

Comment:
yes, that's great. Please add in all docstrings with `condition`

Reply:
Alright, I added it just here. I added it only here, as I think for loss we can strictly require a specific shape. 

Whereas for the others we should be more flexible (broadcasting). I will try to make this more clear.